src/CareTogether.Api/appsettings.Development.json

```diff
@@ -42,5 +42,9 @@
   "Research": {
     "ApiKey": "aaaaaaaa-aaaa-aaaa-aaaa-aaaaaaaaaaaa",
     "OrganizationId": "11111111-1111-1111-1111-111111111111"
+  },
+  "GlobalApiAccess": {
+    "ApiKey": "3f121e2f-16e8-43b7-8d6c-e58b11e91362",

```

Comment:
Can you use an all-letters API key so it's clear that this is a test value? Something like `eeeeeeee-...`